MAINTAINERS.md

```diff
@@ -53,3 +53,25 @@ It expects that it will receive the paths to the API JSONs, which should be in t
 If there are multiple API JSONs, which depend on each other(e.g `@ui5/webcomponents` depends on `@ui5/webcomponents-base` for some types),
 it will combine then and return the single `ComponentData` object per component. Additionally resulted `ComponentData` may have reference
 to other `ComponentData` objects.
+
+### Versioning
+
+Versioning is done using Lerna and it is configured to use fixed versioning. This means that all packages will have the same version,
+no matter if they have changed or not. This is done in order to avoid having different versions of the packages in the same project.
+All the versioning is done by the CI, so you don't have to worry about it, but if you want to bump the version manually, you can do it
+by running `lerna version` command, how you like it and push the changes to the main or create a PR with a version bump and title
+`chore(release)`. CI will release that version when it is merged to the main.
+
+Releasing RC is done automatically, after every push to the main(merges included).
+
+Releasing stable versions is done manually from GitHub action workflow dispatch of workflow `Create Release` or by running `lerna version` command and pushing the changes.
+
+Hotfixes are done in the same way, check out the tag version you want to bump, then add your changes and then just run
```

Comment:
About this part we need to maybe discuss again. For what purpose would we create hotfixes? Wouldn't the approach of having major.minor version branches properly handle this? Maybe you can elaborate a bit more on this...